diff --git a/dist/index.d.mts b/dist/index.d.mts
index 7fd2e84f20f1354014b618256d48e4fb92cfce95..42b37297c4b265f16f14b313b99cc8b696f0dc40 100644
--- a/dist/index.d.mts
+++ b/dist/index.d.mts
@@ -4,7 +4,7 @@ export { AuthorizationScope, GetAuthorizationUrlOptions, PKCEExtensionOptions, S
 import * as Context from 'effect/Context';
 import * as Effect from 'effect/Effect';
 import * as Layer from 'effect/Layer';
-import { HttpClient } from 'effect/unstable/http';
+import { HttpClient } from 'effect/http';
 
 interface BrowserRefreshableTokens {
     readonly accessToken: string;
diff --git a/dist/index.d.ts b/dist/index.d.ts
index 7fd2e84f20f1354014b618256d48e4fb92cfce95..42b37297c4b265f16f14b313b99cc8b696f0dc40 100644
--- a/dist/index.d.ts
+++ b/dist/index.d.ts
@@ -4,7 +4,7 @@ export { AuthorizationScope, GetAuthorizationUrlOptions, PKCEExtensionOptions, S
 import * as Context from 'effect/Context';
 import * as Effect from 'effect/Effect';
 import * as Layer from 'effect/Layer';
-import { HttpClient } from 'effect/unstable/http';
+import { HttpClient } from 'effect/http';
 
 interface BrowserRefreshableTokens {
     readonly accessToken: string;
diff --git a/dist/index.js b/dist/index.js
index 69607fbc03eabe32598174ea5a9800f712178bc4..f5f7c7195f9db695a36e61f84d64d38db3ba480b 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -42,7 +42,7 @@ module.exports = __toCommonJS(index_exports);
 var Context = __toESM(require("effect/Context"));
 var Effect2 = __toESM(require("effect/Effect"));
 var Layer = __toESM(require("effect/Layer"));
-var import_http = require("effect/unstable/http");
+var import_http = require("effect/http");
 var import_core = require("@spotify-effect/core");
 var import_core2 = require("@spotify-effect/core");
 
diff --git a/dist/index.mjs b/dist/index.mjs
index ec0a5d5cb617037490c86eb23323036c34e51369..c2274c064db8671143710c26a5e8cf629a7c465f 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -2,7 +2,7 @@
 import * as Context from "effect/Context";
 import * as Effect2 from "effect/Effect";
 import * as Layer from "effect/Layer";
-import { FetchHttpClient, HttpClient } from "effect/unstable/http";
+import { FetchHttpClient, HttpClient } from "effect/http";
 import {
   Albums,
   Artists,
